stable/weave-scope/charts/weave-scope-agent/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 description: A Helm chart for the Weave Scope cluster visualizer node agent.
 name: weave-scope-agent
-version: 1.1.5
+version: 1.11.6
 appVersion: 1.11.5
```

Comment:
Could you please make this the same as in `weave-scope/Chart.yaml` above, just to make it consistent for now?

```
version: 1.1.6
appVersion: 1.11.6
```